src/documents/models.py

```diff
@@ -872,3 +875,10 @@ class Meta:
 
     def __str__(self):
         return f"{self.name}"
+
+
+if settings.AUDIT_ENABLED:
+    auditlog.register(Document, m2m_fields={"tags"})
+    auditlog.register(Correspondent)
+    auditlog.register(Tag)
+    auditlog.register(DocumentType)
```

Comment:
What about notes?

Reply:
This is entry for adding a note, as you can see its not really helpful
<img width="771" alt="Screenshot 2023-10-22 at 11 04 26 PM" src="https://github.com/paperless-ngx/paperless-ngx/assets/4887959/72393240-7797-4a2c-99fc-3f806ec8ea49">

Reply:
Hmm yes I see that as well. It looks like the "problem" is that the notes are not part of the document model but completely seperate. I haved add the notes log to the model so the changes are noted there correctly. But since there is no note filed in the documents model I have to see about it